include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1884,6 +1884,24 @@ class MatrixFree : public Subscriptor
   get_face_quadrature(const unsigned int quad_index         = 0,
                       const unsigned int hp_active_fe_index = 0) const;
 
+  /**
+   * Return the category the current batch range of cells was assigned to.
+   * Categories run between the given values in the field
+   * AdditionalData::cell_vectorization_category for non-hp-DoFHandler types
+   * and return the active FE index in the hp-adaptive case.
+   */
+  unsigned int
+  get_cell_range_category(
```

Comment:
@Rombur I guess you use `get_cell_category`? Or how do you skip cells with `FENothing`?

Reply:
I use `create_cell_subrange_hp_by_index` to get the cells that have a `fe_index` different than the one I use for `FE_Nothing`